packages/scripts/scripts/check-licenses.js

```diff
@@ -0,0 +1,208 @@
+/**
+ * External dependencies
+ */
+const spawn = require( 'cross-spawn' );
+const { existsSync, readFileSync } = require( 'fs' );
+const chalk = require( 'chalk' );
+
+/**
+ * Internal dependencies
+ */
+const { hasCliArg } = require( '../utils' );
+
+const ERROR = chalk.reset.inverse.bold.red( ' ERROR ' );
+
+const prod = hasCliArg( '--prod' ) || hasCliArg( '--production' );
+const dev = hasCliArg( '--dev' ) || hasCliArg( '--development' );
+const gpl2 = hasCliArg( '--gpl2' );
+
+const gpl2Licenses = [
+	'Apache-2.0 WITH LLVM-exception',
+	'Artistic-2.0',
+	'BSD',
+	'BSD-2-Clause',
+	'BSD-3-Clause',
+	'BSD-like',
+	'CC-BY-3.0',
+	'CC-BY-4.0',
+	'CC0-1.0',
+	'GPL-2.0',
+	'GPL-2.0+',
+	'GPL-2.0-or-later',
+	'ISC',
+	'LGPL-2.1',
+	'MIT',
+	'MIT/X11',
+	'MIT (http://mootools.net/license.txt)',
+	'MPL-2.0',
+	'Public Domain',
+	'Unlicense',
+	'WTFPL',
+	'Zlib',
+	'(MIT AND BSD-3-Clause)',
+	'(MIT AND Zlib)',
+];
+
+const ossLicenses = [
```

Comment:
What does this variable mean? Aren't all of these licenses open source software licenses? 😄

Reply:
THESE ARE NOT THE FREE-EST LICENSES MATT

RICHARD STALLMAN IS JUDGING US FOR EVEN MENTIONING THEM

😁

Reply:
I CLEARLY HATE FREEDOM, MY MICROWAVE IS RUNNING APACHE 2 FIRMWARE.